### ISSUE TITLE
Assigning text to a `Label` with `wordWrap = true` will not update `height` property immediately (/even with `validate()`)

### ISSUE THREAD

**Issue by 2jfw** [BowlerHatLLC/feathers-sdk#58]
I'm assinging text to a `Label` which has set `wordWrap = true`.
Even when calling `validate()` after assigning the text , the height of the label will not properly state the actual height.
Making a delayed call to trace the height of the label, e.g. by `setTimeout` of a very short duration (like 10ms) will return the correct height.

Is there any method to call or any tip on how to force an immediate update of the label's height?

**Comment by joshtynjala** [comment]
@2jfw Did you set the `width` of the `Label` before calling `validate()`?

**Comment by 2jfw** [comment]
I did not, the `Label`'s text was assigned **before** it had a width.

I changed the order on how the related ItemRenderer processes and displays it's data (/ `draw` -> `layoutChildren` -> `commitData`) and now it's working without the need of a delayed call to get the correct height!

[ItemRenderer may require additional optimization but at least I could get rid of the `setTimeout` !]

Many thanks (and also again for your very quick response)!
